grpc/src/main/java/com/linecorp/armeria/server/grpc/FramedGrpcService.java

```diff
@@ -243,6 +243,10 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest req) throws
                                     ctx, defaultHeaders.get(serializationFormat).toBuilder(),
                                     GrpcStatus.fromThrowable(statusFunction, ctx, e, metadata), metadata));
                 }
+            } else {
+                // As per gRPC specification, if timeout is omitted a server should assume an infinite timeout.
```

Comment:
Question) do we set the deadline for client-side? (I couldn't find any)
To rephrase, if a user just creates an armeria gRPC client/server by default, will no timeouts be applied?

Reply:
> the default client-side timeout of a response in milliseconds. Default: 15000

From https://armeria.dev/docs/client-timeouts/

Reply:
Oops, I think I misread the code the first time I reviewed 😅   Double checked that we're setting the header 👍 

https://github.com/line/armeria/blob/863e27c5268631360c14e50e13cd3e38d84c9db5/grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java#L237-L241

Reply:
If a deadline is specified in `CallOptions`, the deadline value is used in the code block you linked. spechttps://github.com/line/armeria/blob/3595dacd9e7675fec309a1029a2a97b829c08682/grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientTest.java#L1466-L1466